compiler-core/src/ast.rs

```diff
@@ -2336,7 +2336,10 @@ pub enum AssignmentKind<Expression> {
     Generated,
     /// let assert x = ...
     Assert {
+        /// The src byte span of the `let assert`
```

Comment:
It's not really clear if this spans just the two keywords or the entire expression, maybe we could add an example?
```
/// ```gleam
/// let assert Wibble = todo
/// ^^^^^^^^^^^^^^^^^^^^^^^^ This
/// ```
```
If it is indeed the entire expression